test/unit/src/common/components/select-repository-list/t_select-repository-list.js

```diff
@@ -95,7 +96,7 @@ describe('<SelectRepositoryListComponent /> instance', function() {
     });
 
     it('should show not message about 0 selected repos', function() {
-      expect(wrapper.html()).toNotInclude('0 selected of');
+      expect(wrapper.html()).toNotInclude('0 selected,');
```

Comment:
I think we shouldn't just check if "0 selected" doesn't show up. The rest ("," or "of") depends on search which is separate feature and it would be best not depending this test on it.

So: `expect(wrapper.html()).toNotInclude('0 selected');`